### DISCUSSION TITLE
Working much better after HF1

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#11134] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

After having issues with services not existing on Receiver nodes, services not starting on sensor nodes and NOTHING ever appearing in the databases I decided to blow away my 2.4 lab systems and start again.
I build a distributed manager using 2.4.10, then left it alone for 12 hours to finish building in the background.  I then used soup to install HF1.
I added two search nodes, which joined the grid with no issues.
I then built a physical sensor using hardware that was running 2.3.260 find, but would not work with 2.4.10, but this time it built and joined the grid with no issues, and events starting appearing in the database and on the alerts screens.
I added a receiver node, and it joined the grid first time (BIG improvement from the past experiences).
I now have a running 2.4.10 grid in my lab, all talking to each other, and seeing network traffic and generating alerts.
Thanks for the great work guys!!!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Great, glad it's working well for you!
